ramlfications/parser/types.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015 Spotify AB
```

Comment:
I'm _pretty_ sure you're not a Spotify employee :) I think, since this is a new module, this should be `2016` and your name instead of Spotify.